bundle/config/scripts.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+type Command string
+type ScriptHook string
+
+const (
+	ScriptPreInit    ScriptHook = "preinit"
+	ScriptPostInit   ScriptHook = "postinit"
+	ScriptPreBuild   ScriptHook = "prebuild"
+	ScriptPostBuild  ScriptHook = "postbuild"
+	ScriptPreDeploy  ScriptHook = "predeploy"
+	ScriptPostDeploy ScriptHook = "postdeploy"
```

Comment:
Please add a "these hook names are subject to change and currently experimental" comment.

Reply:
Or move them to `experimental.go`.